src/docserv/docserv.py

```diff
@@ -308,6 +308,10 @@ def parse_config(self, argv):
                 self.config['targets'][config[section]['name']]['build_container'] = False
                 if 'build_container' in list(config[section].keys()):
                     self.config['targets'][config[section]['name']]['build_container'] = config[section]['build_container']
+
+                self.config['targets'][config[section]['name']]['site_sections'] = config[section]['site_sections']
+                # self.config['targets'][config[section]['name']]['default_site_sections'] = config[section]['default_site_sections']
```

Comment:
default_site_section should be in singular -- it is supposed to be responsible for deciding which file to rename to `index.html`. Doing that for multiple files makes no sense, they'd just overwrite each other.

Reply:
Ahh, good catch. It's corrected in commit 52e7cb8.